vulnerability-disclosure-policy.md

```diff
@@ -68,26 +68,10 @@ Please keep your vulnerability reports current by sending us any new information
 
-This public disclosure policy applies to all submissions under the TTS Vulnerability Disclosure Policy, including Duplicates, Out of Scope, and Not Applicable submissions.
+TTS is committed to patching vulnerabilities within 90 days or less, and disclosing the details of those vulnerabilites once patches are available. We believe that public disclosure of vulnerabilities is an essential part of the vulnerability disclosure process, and that one of the best ways to make software better is to enable everyone to learn from each other's mistakes.
 
-#### Nondisclosure
+At the same time, we believe that disclosure in absence of a readily available patch tends to increase risk rather than reduce it, and so we ask that you refrain from sharing your report with others while we work on our patch.  If you believe there are others that should be informed of your report before the patch is available, please let us know so we can make arrangements.
 
-For 90 days, TTS's default submission disclosure state is nondisclosure for 90 days after submission. Within this 90 day time period, no submissions may be publicly disclosed in full or part without explicit permission from TTS.
-
-During nondisclosure, TTS may patch or update our open source code in response to a submission. This is not public disclosure.
- 
-#### Coordinated Disclosure by TTS
-
-After resolving your submission, we may coordinate with you to disclose your report. By default, we will disclose in full all communications between us from first report to resolution.  We believe that public disclosure of vulnerabilities is a healthy and important part of the vulnerability disclosure process and that sharing found vulnerabilities to teach others helps prevent the same vulnerabilities from being reproduced elsewhere.
-
-If warranted by the specific situation, there are two ways we might limit the information shared: redacting sensitive information or limiting visibility to a summary and partial timeline written by us with input from you.
-
-In general, reports will be published on HackerOne or via a Github issue. We will not publish communications with you or information about you without your permission.
-
-#### Disclosure by you
-
-After the nondisclosure period, TTS welcomes self-disclosure of your work. You may also immediately republish information publicly disclosed by TTS. We may ask, but will not require, that you redact specific details of your work.
-
-Do not publish undisclosed communications or information from TTS personnel without permission from TTS.
+Once the patch is available, we may want to coordinate an advisory with you, but you are also welcome to self-disclose if you prefer.  By default, we prefer to disclose everything, but we will never publish information about you or our communications with you without your permission.  In some cases, we may also have some sensitive information that should be redacted, and so please check with us before self-disclosing.
```

Comment:
> we will never publish information about you or our communications with you without your permission.

This is a little stronger than I think we want. I don't think a reporter should expect that they can prevent us from disclosing a vulnerability, either publicly or to our partners. I think it would be better to emphasize that we prefer to disclose everything and may share reported vulnerabilities with other agencies or with the public, though we will notify the reporter before doing so, so they have a chance to request redactions or changes.

Reply:
@konklone The scope here is not "the vulnerability".  The scope here is "we fixed this vulnerability because John Doe sent us this here report, described in his words as..." .  That's something I believe warrants asking for permission, without preventing us from disclosing a vulnerability.